models/psf.py

```diff
@@ -527,7 +527,44 @@ def get_upstreams(self, session=None):
         with SmartSession(session) as session:
             return session.scalars(sa.select(Image).where(Image.id == self.image_id)).all()
         
-    def get_downstreams(self, session=None):
-        """Get the downstreams of this PSF (currently none)"""
-        return []
+    def get_downstreams(self, session=None, siblings=False):
+        """Get the downstreams of this PSF.
+
+        If siblings=True (default) then also include the PSFs, WCSes, ZPs and background objects
+        that were created at the same time as this source list.
+        """
+        from models.source_list import SourceList
+        from models.world_coordinates import WorldCoordinates
+        from models.zero_point import ZeroPoint
+        from models.provenance import Provenance
```

Comment:
I don't think this helps in terms of performance. 

This is the sort of place where if you include all the imports on each one of the classes, you'd get a circular import. 
Usually when I setup the imports there's a clear hierarchy: for example a SourceList knows about an Image but not the other way around (Image is more "basic"), so we can import Image at the start of SourceList. 

But when they need to know about each other, there has to be an import call "when needed" inside a function to avoid the circular import at startup. By the time you call this function, there's a good chance that all the modules are already loaded so it doesn't do anything except put the names of classes into local scope.